modules/pytket-braket/docs/changelog.rst

```diff
@@ -1,6 +1,11 @@
 Changelog
 ~~~~~~~~~
 
+0.16.0 (February 2022)
+----------------------
+
+* Updated pytket version requirement to 0.19.
```

Comment:
Should this include the change in the gate set?

Reply:
Will add a note about this.